app/models/user.rb

```diff
@@ -138,6 +134,33 @@ def self.to_csv(options = {}, timezone = 'UTC')
       end
     end
   end
+  
+  def self.find_from_potential_ride( potential_ride )
+    user = User.find_by_email( potential_ride.email )
+    unless potential_ride.phone_number.empty? || potential_ride.phone_number_normalized == '+15555555555'
```

Comment:
Perhaps `'+15555555555'` belongs in something like `User::DUMMY_PHONE_NUMBER`

Reply:
good suggestion, done